neptune/new/sync/utils.py

```diff
@@ -162,11 +162,25 @@ def get_offline_runs_ids(base_path: Path) -> List[str]:
         return []
     for run_path in (base_path / OFFLINE_DIRECTORY).iterdir():
         dir_name = run_path.name
-        run_id = dir_name[len("run__") :]
+        match = re.match(CONTAINER_DIR_REGEXP, dir_name)
+        if match is None:
+            raise ValueError(f"Wrong container directory name: {dir_name}")
+        run_id = match.group(3)
         result.append(run_id)
     return result
 
 
+def get_offline_run_dir_path(base_path: Path, run_id: UniqueId):
+    for dir_ in [
+        base_path / OFFLINE_DIRECTORY / run_id,
+        base_path / OFFLINE_DIRECTORY / f"run__{run_id}",
```

Comment:
OK, I think now you're purposefully writing broken code which will have to be removed around next PR.

You could just list directories in `offline` and try to recognize them as container IDs (exactly as it is done for `sync`??), yet you do a "hit-and-miss" strategy of checking for hypothetical dirnames. Why?

Reply:
I've mentioned this problem [here](https://github.com/neptune-ai/neptune-client/pull/817#discussion_r793751495).

But as we've finally agreed on.
Right now I'm listing and accepting parameters which are indentical as directory in which queue is stored.

So user has to pass full directory name: `run__<uuid>` (except situation with depracted run format without type prefix).
I don't especially asume `run_` prefix anywhere, but in the end if  prefix doesn't match to any existing `ContainerType` nor is `ContainerType.run` the program will finally crash.

@pkasprzyk right?